client/src/Components/Coins.js

```diff
@@ -1,24 +1,22 @@
 import React from 'react';
 import styled from 'styled-components';
-import StarNotFilled from '../images/star.svg';
-
-export default function Coins({
-  key,
-  name,
-  image,
-  symbol,
-  marketCap,
-  price,
-  priceChange,
-  volume,
-}) {
+import { ReactComponent as StarNotFilled } from '../images/star-noFill-empty.svg';
+import { ReactComponent as StarFilled } from '../images/star-noFill-filled.svg';
+
+export default function Coins({ coin, onToggleFavorite }) {
+  const { image, name, symbol } = coin;
+  const priceChange = coin.price_change_percentage_24h;
+  const price = coin.current_price;
+  const marketCap = coin.market_cap;
+  const volume = coin.total_volume;
```

Comment:
For me this could be a bit cleaner. It doesn't have to be one line but maybe three are enouth?

```suggestion
export default function Coins({ coin, onToggleFavorite }) {
  const { image, name, symbol } = coin;
  const {price_change_percentage_24h: priceChange, current_price: price} = coin;
  const {market_cap: marketCap, total_volume: volume} = coin;
```